### ISSUE TITLE
Idea: Burlap Balance

### ISSUE THREAD

**Issue by PearlCanopus** [squeek502/VeganOption#84]
In conclusion I could say: Make jute stalks less rare (biomes) or create an own jute plant. But give it also less drop per plant. Make the refining process longer and a bit more complex (maybe configurable easy or hard). Add a further step to create leather from burlap. Add recipes for bags of burlap. With "magical" mods added, those bags could be magically expanded.

complete idea:
I know you need some time to search a biome which contains large fern. This might be a bit annoying. To take seed from them is a good step. Maybe there should be more options, like configurable seed drop from braking tall grass.

But if you've found a matching biome burlap becomes op over leather.

My Idea:
- wider availability of large fern seeds
- lower output of jute stalks per drop (0-1) when harvested by fist
- 1-2 drops if harvested with shears
- 2-5 drops if harvested with looting enchanted or luck modified shears (L3)
Else you may align harvesting with rightclick for 1-2 jute fiber per fern.

still 3x3 stalks for jute block

- jute block needs 3x more time to rot
or
- rotting may work different for biomes (temperature and humidity) with configurable multipliers i.e.
Swamp: x 1.0
Jungle: x 1.5
Plaines/Forest: x 2.5
Icy: x 5.0
Desert/Oasis: x 1.3
Mountains: x 3.0

Breaking the blocks by fist will drop a bit less jute fibre.
Maybe using (enchanted/upgraded) shears or sickles may rise the outcome to actual values.

Possibly you can insert drying, if drying racks are available from mods or melting/pressing instead. A block for manual pressing would be ok too, if mod is used standalone. A presser could be used for other recipes too.

If used in ocean/beach biomes and dryed regularily, it could drop salt or saltpeter (if any mod adds it).

Burlap could be used for bags and cloth but not necessarily as substitution of leather.
In order to get leather from burlap, you should craft it together with wax (from any mod) or plant/cooking/creosole oil (oil cotton) additionally.
There is Candelilla out there (don't know which mod adds it). Harvestcraft provides wax berries and soya (soya wax could be a vegan recipe from soya oil)...

Maybe you can add config option for easy (as always) or complex/hard with some more options and alternative drops and maybe experience.

**Comment by elifoster** [comment]
>But if you've found a matching biome burlap becomes op over leather.

I'm not sure if I'd agree. I haven't played Minecraft in a long time but from what I remember cows are pretty widely available particularly once you find a biome they are common in (I'm not sure what they are common in specifically but I remember it being the same biomes that you find the stuff that drops jute).

>jute block needs 3x more time to rot

Despite being a GregTech player I actually find time based balance to be quite boring. It doesn't make anything more challenging, it just means you have to wait longer. Like, waiting in line at the grocery store doesn't make that grocery trip more exciting or interesting just because it took longer.

>rotting may work different for biomes (temperature and humidity) with configurable multipliers 

Biome specific retting would only really make sense if we were doing [dew retting instead of water retting](https://en.wikipedia.org/wiki/Retting).

>Possibly you can insert drying, if drying racks are available from mods or melting/pressing instead. A block for manual pressing would be ok too, if mod is used standalone. A presser could be used for other recipes too.

I don't really understand this idea. Could you rephrase it?

>Burlap could be used for bags and cloth but not necessarily as substitution of leather.
In order to get leather from burlap, you should craft it together with wax (from any mod) or plant/cooking/creosole oil (oil cotton) additionally.
There is Candelilla out there (don't know which mod adds it). Harvestcraft provides wax berries and soya (soya wax could be a vegan recipe from soya oil)...

Is there a real world example of burlap being used to make an even stronger fabric? I'm not familiar with any uses of it like that.

>Maybe you can add config option for easy (as always) or complex/hard with some more options and alternative drops and maybe experience.

Like what?

**Comment by PearlCanopus** [comment]
I checked out a Mega Taiga biome and walked around about half an hour. I never needed to harvest it again. With conveyor belts in game I could need that more often, I think. It's strongly depending on the leather sinks. :)
Leather from cows I have to harvest again and again and I have to wait they'll grow up, because they do not respawn frequently, if I have no animal farm. So far it could be challenging, if you have no fully automated farm with breeder, feeder and slaughterhouse. But this is no early game solution I think. :)

To automate large fern is harder or nearly impossible. It could become easier, if the upper part of the plant has to be harvested like IE hemp.

OK, I cannot argue against time based balance. It sounds good, when I wrote it down. I play very chilled and have no problem to wait a bit longer for stuff, because  do antything else meanwhile. ;) But it's ok, Minecraft trees also grow up in one day. So far... ;)

When I talked about water retting I meaned water temperature. It's a biological effect which will take much longer, if it's cold. But this is a more realistic approach, no need of curse. We not really have climate in minecraft (if I don't use climate mods). So far that's not important to me.

I thought about drying after harvesting jute fibre from under water.
Some mods (i.e. Tinkers Construct) add drying racks to do that (maybe stackwise to fasten up).
But you could also press water out of the fibres instead of drying them...
If the last option would be preferred, Vegan Option could add a manual presser or add a recipe for Harvestcraft presser to do this.
If you'd decide to add a block like manual presser it could be used without any energy cranked by fist. (low tech)
Sure, both won't be necessary but sounds cool for me.

Oil Cotton is a real world alternative for leather. I don't know whether anybody tried that with burlap yet. :)
The oil within the fibres impregnate them. So the coat becomes tear- and waterproof. My friend got a coat like this from an Australian Outback distibutor. It feels like leather.

**Comment by squeek502** [comment]
I might be out of touch, but since when was leather important to balance/progression? Last I played (it has been a while...), leather was just a side ingredient for a few fairly one-off/niche recipes and you rarely needed a steady supply of it.

Or, to put that another way, what's the harm in burlap/leather being too easy to get? I feel like this issue is missing a description of the problem.

**Comment by PearlCanopus** [comment]
Vanilla Minecraft has only a few recipes with leather. That's true.
I watched it from a modded Minecraft standpoint. TVO would rarely be a standalone mod for the player. ;)
But I don't want bother you. You may close this issue.

There is no problem with the availability of burlap.
I only mentioned that getting it too easy would look like cheating or op compared to vanilla minecraft (if each burlap is equal to one leather).
Leather is used by not only niche tec mods for item transport (i.e. conveyor belts).
Harvestcraft introduce the Hardened Leather Armor (waxed leather).
Backpacks often use leather.
There might be other mods out there using leather in recipes, which I don't use/know.

**Comment by squeek502** [comment]
Yeah, but as @elifoster said it's not like vanilla leather is hard to get, either--in fact, it's easier than burlap (killing a cow is the beginning and end of the process). Plus, the availability of ferns can vary *a lot*, and it's something that's pretty much out of TVO's control (modded biomes could make ferns extremely plentiful, vanilla jungles might happen to be hard to find, etc).

Closing for now, I think Jute/Burlap is pretty good where it's at.
